bundle/edu.gemini.ags.client.impl/src/main/scala/edu/gemini/ags/client/impl/AgsHttpClient.scala

```diff
@@ -19,9 +24,12 @@ object AgsHttpClient {
   private val hostnameVerifier: HostnameVerifier = new HostnameVerifier {
      def verify(s: String, sslSession: SSLSession) = true
   }
+
+  // LRU cache to minimize lookups.
+  val lRUCache = Collections.synchronizedMap(new LruCache[String, AgsResult.Success](100))
```

Comment:
I might bump this up some more.  It wouldn't take that much space if it were, say, 1000 entries.  There are some pretty long target lists in some proposals.